### PR TITLE
update import

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

File: src/ontology/imports/merged_import.owl [hunk 1]
```diff
@@ -347658,21 +348715,17 @@ Classes for population already  exist in IDO (&apos;organism population&apos;, I
...
-            <owl:Class>
-                <owl:intersectionOf rdf:parseType="Collection">
-                    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/UBERON_0001032"/>
-                    <owl:Restriction>
-                        <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/BFO_0000050"/>
-                        <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/UBERON_0000012"/>
-                    </owl:Restriction>
-                </owl:intersectionOf>
-            </owl:Class>
-        </owl:equivalentClass>
```
Comment (matentzn) [inline src/ontology/imports/merged_import.owl:347671]: @shawntanzk do you see what I mean?

Comment (shawntanzk) [reply src/ontology/imports/merged_import.owl:347671]: yup this removal should solve it :) thanks!

**Review by matentzn — COMMENTED** [review]
Despite https://github.com/geneontology/go-ontology/issues/23916 I will merge this..

File: src/ontology/imports/merged_import.owl [hunk 2]
```diff
@@ -60113,7 +60551,7 @@ For example, A and B may be gene products and binding of B by A positively regul
                 <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/GO_0002775"/>
             </owl:Restriction>
         </owl:annotatedTarget>
-        <obo1:IAO_0000233>https://github.com/geneontology/go-ontology/issues/13926</obo1:IAO_0000233>
+        <obo2:IAO_0000233>https://github.com/geneontology/go-ontology/issues/13926</obo2:IAO_0000233>
```
Comment (matentzn) [inline src/ontology/imports/merged_import.owl:60554]: @anitacaron check this - it is used.. Just as a curie.

**Pull request merged**